File: C++/project_04.cpp
// Temperature Converter (Celsius ↔ Fahrenheit)

#include <iostream>
using namespace std;
int main()
{
    int choice;
    float celsius, fahrenheit;
    cout << "___MENU___" << endl;
    cout << "1. Enter your Celsius and convert to  Fahrenheit" << endl;
    cout << "2. Enter your Fahrenheit and convert to Celsius" << endl;
    cout << "choice 1 and 2 : ";
    cin >> choice;

    if (choice == 1)
    {
        cout << "Enter your Celsius : ";
        cin >> celsius;
        cout << "Your Fahrenheit is : " << ((celsius * 9 / 5) + 32);
    }
    else if (choice == 2)
    {
        cout << "Enter your Fahrenheit : ";
        cin >> fahrenheit;
        cout << "Your Celsius is : " << ((fahrenheit - 32) * 5 / 9);
    }
    else
    {
        cout << "Invalid choice please enter 1 and 2.";
    }

    return 0;
}

File: Python/project_04.py
# Temperature Converter (Celsius ↔ Fahrenheit)

print("         ___MENU___")
print("1 Convert Celsius to  Fahrenheit ")
print("2 Convert Fahrenheit to  Celsius ")
choice = int(input("Press 1 and 2 : "))

if(choice == 1):
    celsius = float(input("Enter your Celsius : "))
    print("Your Fahrenheit is : ",((celsius * 9/5) + 32))
elif(choice == 2):
        fahrenheit = float(input("Enter your Fahrenheit : "))
        print("Your Celsius is : ",((fahrenheit - 32) * 5/9))
else:
      print("Invalid choice! Please enter 1 or 2.")